UdpSockets.m

```diff
@@ -82,6 +82,20 @@ @implementation UdpSockets
     if (callback) callback(@[]);
 }
 
+RCT_EXPORT_METHOD(setBroadcast:(NSString*)cId
+                  flag:(BOOL)flag
+                  callback:(RCTResponseSenderBlock)callback) {
+    UdpSocketClient* client = [self findClient:cId callback:callback];
+    if (!client) return;
+
+    NSError *error = nil;
+    if (![client setBroadcast:flag error:&error])
+    {
+        callback(@[error]);
```

Comment:
makes sense, node dgram sockets also throw an error on setBroadcast before bind. 

Would you mind making that explicit? Like checking if this._state === STATE.UNBOUND and throwing an error, something like "you must bind before setBroadcast". Because that red box error is pretty inscrutable